app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+</div>
+  
+  
+  <div class="field mb-3">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field mb-3">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field mb-3">
+    <%= f.label :password_confirmation ,class:'form-label'%><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up",class:'btn btn-secondary' %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] You still need to modify your sign-up view to be centered just as you have done with the log-in view

  ![sign_up](https://user-images.githubusercontent.com/81468675/188130557-f0653a9b-d2e2-4f95-afef-363dbea1020f.jpg)

Reply:
hello @julie-ify, thank you for your review, I was going to do this at the end once my application was functionality was okay, but since you have brought this up now, I will do it, even though this was not a requirement for this milestone. I would have appreciated this to be an optional change instead. thank you :pray: